src/Frontend/test/util/react/FormValueContainerSpec.js

```diff
@@ -22,20 +22,6 @@ describe('FormValueContainer', () => {
   it('can set/get form values', () => {
     instance.persistFormValue('prefix.value', 'blah');
     expect(instance.getFormValueForAlias('prefix.value')).to.equal('blah');
-  });
-
-  it('can purge form values', () => {
-    instance.persistFormValue('prefix.value1', 'blah');
-    instance.persistFormValue('prefix.value2', 'blah');
-
-    expect(typeof instance.getFormValueForAlias('prefix.value1')).to.not.equal('undefined');
-    expect(typeof instance.getFormValueForAlias('prefix.value2')).to.not.equal('undefined');
-
-    instance.purge('prefix');
-
-    setTimeout(() => {
-      expect(typeof instance.getFormValueForAlias('prefix.value1')).to.equal('undefined');
-      expect(typeof instance.getFormValueForAlias('prefix.value2')).to.equal('undefined');
-    }, 100);
+    localStorage.removeItem('prefix.value');
```

Comment:
mocking localstorage to cover this case should be verified. The whole testcase was quite fragile, but without the explicit localStorage API it might work